docs/faq.rst

```diff
@@ -1,6 +1,20 @@
 FAQ
 ===
 
+What happens if I rename my GitHub repository?
+----------------------------------------------
+
+If you have registered a tool or workflow from GitHub in Dockstore the
+link to the repository for your tool or workflow is displayed on the
+Info tab next to 'Source Code'. If you then `rename <https://docs.github.com/en/github/administering-a-repository/renaming-a-repository>`__
+the GitHub repository, the Source Code link will be incorrect, but
+will resolve to the correct GitHub location when you click on it.
+
+Another side effect is that you will be able to register the workflow
```

Comment:
If you reuse the repo name, you're in bigger trouble. But GitHub itself councils against it.

Reply:
Yeah, could be worth warning that you normally want to avoid this.

Reply:
Added warning statement